demos/basic/dune

```diff
@@ -0,0 +1,9 @@
+(executable
+ (name basic)
+ (public_name opencv.basic)
```

Comment:
Won't giving it a `public_name` mean that the executable will be installed under Opam's `bin` directory? I am not sure if that was the intention with these demos.

Reply:
Indeed. I will remove the public name